### ISSUE TITLE
revisit indexing of Publication Status facet (publicationStatus Solr field)

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#2377]
In Solr we index an array of "publicationStatus" values which can have multiples such as both "Draft" and "Unpublished" but we don't index "Deaccessioned" as one of these values.

We did things this way because of a decision in summer 2014 to "Show a 'Deaccessioned' label on the dataset card but don't show a facet" and there is some lively debate in comments: https://docs.google.com/document/d/1lFcwjtdGIqqQLYYnwtTI2GZNzmzYlrnMk2omGcXO9oY/edit?usp=sharing

We're thinking about revisiting this decision. At minimum we think we want to start indexing "Deaccessioned" as a possible value for "publicationStatus" so it can be operated on by MyData (#2302 says "Add in review and deaccession statuses for datasets"). (In review is a whole discussion in and of itself.)

**Comment by pdurbin** [comment]
@eaquigley I made the small change we talked about to start indexing and showing "Deaccessioned" as one of the possible values for "Publication Status" in 824c4b0 . It follows the same rules as "Draft" and "Unpublished" in that the entire "Publication Status" facets is hidden unless there are there are cards that are in an unpublished, draft, or deaccessioned state. To see the change you'll need a build newer than v. 4.1 build 52. I'm giving this to you to talk with @scolapasta @mheppler @mcrosas etc. about what we want. The change I made was small and we can easily back out of it.
